### PR TITLE
Add use_gpu to CLI

### PR DESCRIPTION
**Pull request by rvlb**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -21,11 +21,11 @@ jobs:
...
-    - name: Cache installed requirements
-      uses: actions/cache@v2
-      with:
-        path: ${{ env.pythonLocation }}
-        key: ${{ matrix.os }}-${{ env.pythonLocation }}-${{ hashFiles('**/requirements.txt', '**/requirements-dev.txt') }}
+    #- name: Cache installed requirements
+    #  uses: actions/cache@v2
+    #  with:
+    #    path: ${{ env.pythonLocation }}
+    #    key: ${{ matrix.os }}-${{ env.pythonLocation }}-${{ hashFiles('**/requirements.txt', '**/requirements-dev.txt') }}
```
Comment (rvlb) [inline .github/workflows/ci.yml:28]: https://github.com/pypa/pip/issues/9880#issuecomment-827807619

File: entity_embed/cli.py [hunk 2]
```diff
@@ -366,7 +368,7 @@ def _load_model(kwargs):
         model_cls = EntityEmbed
 
     model = model_cls.load_from_checkpoint(kwargs["model_save_filepath"], datamodule=None)
-    if torch.cuda.is_available():
+    if torch.cuda.is_available() and kwargs["use_gpu"]:
```
Comment (fjsj) [inline entity_embed/cli.py:371]: always force cuda when `use_gpu`

**Comment by fjsj** [comment]
- Please document on docs/guide/cli.rst
- Please add --use_gpu to notebooks/cli-example-generation/*.ipynb
- Please create an issue here on the caching issue, so we can keep track

**Review by fjsj — CHANGES_REQUESTED** [review]
Please check comments.

**Review by fjsj — APPROVED** [review]
LGTM, thanks

**Pull request merged**